<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmCustWOData</class>
 <widget class="QWidget" name="FrmCustWOData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnQuery">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	color:white;
	font: 75 9pt &quot;Agency FB&quot;;
	font: 75 10pt &quot;微软雅黑&quot;;
	background-color:rgb(0, 170, 255);
	border-radius:8px;
}

QPushButton:hover
{
	color:white;
	background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
	color:white;
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
}</string>
        </property>
        <property name="text">
         <string>查询订单</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnModify">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	color:white;
	font: 75 9pt &quot;Agency FB&quot;;
	font: 75 10pt &quot;微软雅黑&quot;;
	background-color:rgb(116, 116, 116);
	border-radius:8px;
}

QPushButton:hover
{
	color:white;
	background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
	color:white;
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
}</string>
        </property>
        <property name="text">
         <string>修改交期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnModify_2">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	color:white;
	font: 75 9pt &quot;Agency FB&quot;;
	font: 75 10pt &quot;微软雅黑&quot;;
	background-color:rgb(100, 150, 1);
	border-radius:8px;
}

QPushButton:hover
{
	color:white;
	background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
	color:white;
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
}</string>
        </property>
        <property name="text">
         <string>清空交期</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>客户PO</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>客户LOT</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="editCustLot">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editCustPO">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>300</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>交期</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="showGroupSeparator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>2020</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>300</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>全选/反选</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="gridStyle">
         <enum>Qt::DashDotLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
